test/specs/landingpage.js

```diff
@@ -0,0 +1,105 @@
+'use strict'
+
+module.exports = { 
+
+    'Testing Pagevamp logo' : function (browser) {
+        browser
+        .url("https://www.pagevamp.com/")
+        .waitForElementVisible("#topmost-nav a.pv-logo",1000)
+        .click("#topmost-nav a.pv-logo")
+        .url(function(result) {
+                this.assert.equal(result.value, "https://www.pagevamp.com/");  //clicking Pagevamp logo
+
+            })
+        .pause(3000)
+        .end();
+    },
+
+    'Testing if "How it works" is working' : function (browser) {
+        browser
+        .url("https://www.pagevamp.com/")
+        .click(".navbar-right li:nth-child(1) a")
+        .url(function(result) {
+                this.assert.equal(result.value, "https://www.pagevamp.com/#how-it-works");   //scroll to How it work section
+                this.assert.containsText("#how-it-works-section","Website in seconds");     //testing "Website in seconds" text
+                this.assert.attributeContains(".link","data-target","#how-it-works-video");  //testing "Watch it in action" link
+            })
+        .pause(3000)
+        .end();
+    },
+
+    
+    'Testing if pricing tab works properly': function (browser) {    
+        browser.url("https://www.pagevamp.com/")
+            .click(".navbar-right li:nth-child(2) a")
+            .url(function (result) {
+                this.assert.equal(result.value, "https://www.pagevamp.com/us/pricing");  //Pricing page url
+                })
+            .waitForElementVisible(".pv-section--center .col-sm-12 .pv-title", 1000)
+            .assert.containsText(".pv-title","Pricing")            // testing if page contains "Pricing" text      
+            .click("#topmost-nav a.pv-logo")
```

Comment:
i think wont need this in every test since its not related with javascript so using the index link would work fine